package.json

```diff
@@ -27,8 +32,6 @@
   "scripts": {
     "lint": "run-p --print-label --continue-on-error lint:*",
     "lint:eslint": "eslint --ignore-path .eslintignore --ext js,jsx,ts,tsx,mjs,cjs,astro .",
-    "lint:ts": "tsc --noEmit",
```

Comment:
no clue why these kept breaking - one failed to parse the `<Fathom>` component's inline script properly, the other failed complaining about missing types for the `.astro` file import :shrug: